src/settings/components/Profile.tsx

```diff
@@ -681,43 +682,34 @@ const Profile: FunctionComponent<
 						<Column size="3-7">
 							<Form type="standard">
 								<>
-									<FormGroup
-										label={t('settings/components/account___voornaam')}
-										labelFor="first_name"
-										error={get(profileErrors, 'first_name')}
-									>
-										<TextInput
-											id="first_name"
-											value={firstName || ''}
-											onChange={setFirstName}
+									<div className="profile-actions">
+										<FormGroup
+											label={t('settings/components/account___voornaam')}
+											labelFor="first_name"
+										>
+											{firstName}
+										</FormGroup>
+										<Button
+											type="secondary"
+											label={t(
+												'settings/components/account___wijzig-gegevens'
+											)}
+											onClick={handleEditInfo}
 										/>
```

Comment:
meemoo wants to use real links as much as possible. So if the url is known before hand, and you're trying to navigate to a page, use a Link element around the button instead of the onclick handler:
```suggestion
										<a href={generateEditProfileInfoLink()}>
											<Button
												type="secondary"
												label={t(
													'settings/components/account___wijzig-gegevens'
												)}
											/>
										</a>
```
That way, users can right click and select "open in new window". Which is something meemoo explicitly asked us to change in the past.